zuul.d/layout.yaml

```diff
@@ -2,10 +2,10 @@
 # zuul.d/layout.yaml
 
 - job:
-    name: molecule-vagrant-fedora
+    name: molecule-vagrant-ubuntu-bionic-py38
     description: Run py38 tox environment
     parent: ansible-tox-py38
-    nodeset: fedora-latest-1vcpu
+    nodeset: ubuntu-bionic-4vcpu
```

Comment:
Please don't do that. fedora and ubuntu are not testing the same thing. Fedora is providing out of the box vagrant/vagrant-libvirt and for ubuntu we need to use the package from hashicorp. It's really important to test non-hashicorp provided version of vagrant. The hashicorp packages are bundling a lot of librairies and are bringing their own set of troubles. Moreover, using fedora latest will test things with recent python/ruby/...

Reply:
The CI doesn't provide the resources required to run this test.

I guess a new nodepool nodeset providing both fedora and 4Go can be added, meanwhile this patch allow to check both py36 and py38 tox environments. It doesn't make sense to keep a broken test: it seems better to use ubuntu rather than disabling this test.